fe/fe-core/src/main/java/com/starrocks/memory/ProcProfileCollector.java

```diff
@@ -46,20 +48,15 @@ public class ProcProfileCollector extends FrontendDaemon {
 
     private long lastCollectTime = -1;
 
-    private boolean initialized = false;
-
     public ProcProfileCollector() {
         super("ProcProfileCollector");
         profileLogDir = Config.sys_log_dir + "/proc_profile";
     }
 
     @Override
     protected void runAfterCatalogReady() {
-        if (!initialized) {
-            File file = new File(profileLogDir);
-            file.mkdirs();
-            initialized = true;
-        }
+        File file = new File(profileLogDir);
+        file.mkdirs();
```

Comment:
check first? no need to mkdir every time?

Reply:
There is check in mkdirs() 
```
    public boolean mkdirs() {
        if (exists()) {
            return false;
        }
        if (mkdir()) {
            return true;
        }
        File canonFile = null;
        try {
            canonFile = getCanonicalFile();
        } catch (IOException e) {
            return false;
        }

        File parent = canonFile.getParentFile();
        return (parent != null && (parent.mkdirs() || parent.exists()) &&
                canonFile.mkdir());
    }
```